### ISSUE TITLE
Is there a predicted code template

### ISSUE THREAD

**Issue by DeerEyre** [meganndare/cantonese-nlp#4]
After running your pre-training model, I find that your work and experiment are really excellent. But after getting your pre-training model, how to translate a single sentence? Is there a corresponding code template

**Comment by meganndare** [comment]
Hi,

You can investigate more in the https://github.com/facebookresearch/UnsupervisedMT repository to achieve this (assuming you are asking about the transformer model?). As a hint, the command may look similar to this:

`python main.py --exp_name model-73-eval --transformer True --n_enc_layers 4 --n_dec_layers 4 --share_enc 3 --share_dec 3 --share_lang_emb True --share_output_emb True --langs 'en,mr' --n_mono -1 --mono_dataset 'en:en.txt.pth,,;mr:mr.txt.pth,,' --para_dataset 'en-mr:,XX_dev.txt.pth,XX_test.txt.pth' --mono_directions 'en,mr' --word_shuffle 3 --word_dropout 0.1 --word_blank 0.2 --pivo_directions 'en-mr-en,mr-en-mr' --pretrained_emb 'shared.embed' --pretrained_out True --lambda_xe_mono '0:1,100000:0.1,300000:0' --lambda_xe_otfd 1 --otf_num_processes 2 --otf_sync_params_every 1000 --enc_optimizer adam,lr=0.00005 --group_by_size True --batch_size 32 --epoch_size 500000 --stopping_criterion bleu_en_mr_valid,16 --freeze_enc_emb False --freeze_dec_emb False --seed 1 --reload_model ./dumped/model-73/as5hcnjn5n/best-em_en_mr_valid.pth --reload_enc True --reload_dec True --eval_only True`

You will need to appropriately substitute the pretrained model location in **--reload_model** and set the **--eval_only** mode to True. Additionally you need to add your own data files at the correct arguments. If you put the sentences you would like to translate into the dev or test parallel dataset args, you could obtain the translations that way and find them in the ./dumped folder. 

Hope this helps!
